### DISCUSSION TITLE
When I use the official mirror 9.1.0, the startup always reports OOM

### DISCUSSION THREAD

**Discussion by wangpf09** [apache/skywalking#9274] [Q&A] [answered]
When I use the official mirror 9.1.0, the startup always reports OOM, and I changed the parameters of -Xms and -Xmx, and it did not solve the problem

![abdd674b074ade3ab3be8695049c707](https://user-images.githubusercontent.com/39894166/175842618-72dc0644-61cd-4234-b86c-f29b30b4324d.png)

**Comment by wu-sheng** [comment, marked as answer]
This looks like a QEMU VM, sorry, that is out of project's scope. All e2e have been run and tested in 3U8G GitHub VM, including ElasticSearch, SkyWalking OAP, and Monitoring service with agents. All checked.

**Reply by wangpf09** [reply]
Ok thanks, I tried to rebuild the image based on the official jar package and changed the base image to `openjdk:11.0.15-jre`. Now it can start normally
